CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/DatabaseRequestHandler.swift

```diff
@@ -298,9 +298,13 @@ public class DatabaseRequestHandler {
         case "database_copy":
             let dbName: String! = args.get(name: "dbName")
             let dbConfig: DatabaseConfiguration? = args.get(name: "dbConfig")
+            let dbPath: String! = args.get(name: "dbPath")
+            try! Database.copy(fromPath: dbPath, toDatabase: dbName, withConfig: dbConfig)
+
+        case "database_getPreBuiltDb":
             let dbPath: NSString! = args.get(name: "dbPath")
             let path: String! = Bundle(for: type(of:self)).path(forResource: dbPath.deletingPathExtension, ofType: dbPath.pathExtension)
-            try! Database.copy(fromPath: path, toDatabase: dbName, withConfig: dbConfig)
```

Comment:
As it is change, is prebuilt database test is verified with this change? If not, can you verify?

Reply:
already done